components/Graph.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect } from 'react';
```

Comment:
`import React from 'react';` is not required with Next.js
```suggestion
import { useEffect } from 'react';
```